apps/bfd-pipeline/bfd-pipeline-rif-extract/pom.xml

```diff
@@ -22,6 +22,13 @@
 			<artifactId>bfd-model-rif</artifactId>
 		</dependency>
 
+		<dependency>
+			<!-- Just useful. -->
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>24.1.1-jre</version>
+		</dependency>
```

Comment:
If we're not already using this somewhere, you'll need to file a SIA for it. Shouldn't be a big deal, though.

You also might consider adding it to the new bfd-shared-utils module, instead, as it definitely is useful, in general.

Reply:
We are already using this for BloomFilters. see: LoadedFilterTest.java - Would this still require an SIA with this being the case?

Reply:
> Would this still require an SIA with this being the case?
Nope!

Reply:
awesome, added it to the bfd-shared-utils